docs/throttling.md

```diff
@@ -1,2 +1,22 @@
-## Throttling
+### Background
+
+We have had to disable network throttling in Synthetics for browser based monitors. This is intended to be a temporary measure until we can release an update in the future to enable it again.
+
+⚠️ please be aware that your monitors may run more quickly (i.e. have a lower duration) with network throttling disabled.
```

Comment:
```suggestion
⚠️ With network throttling being disabled, note that your monitors may run more quickly (i.e. have a lower duration) than what your users would experience in real conditions on average.
```
I took a guess here to complete the sentence and clarify the purpose of this warning, feel free to ignore or correct. We should state why it's important to mention that monitors would run more quickly, what's the consequence?

Reply:
@florent-leborgne thanks for the suggestion.  I'd rather avoid suggesting a link to real user performance, because Synthetics is about a relative measure of performance (where as RUM is more about real user performance), however I appreciate your point.  How about something like this:

> ⚠️ With network throttling being disabled, note that your monitors may run more quickly (i.e. have a lower duration) than you observed previously, and when network throttling is enabled again in the future.

Reply:
Ok this makes perfect sense to me! Thanks

Reply:
@florent-leborgne updated - would you mind giving the PR an approve?